### DISCUSSION TITLE
Drag-and-drop to reorder list elements

### DISCUSSION THREAD

**Discussion by kennydo** [notaapp/talk#365] [Feature Requests]
Hello! I've been a happy Nota user for a while, but a feature that I miss from when I used Notion was being able to drag-and-drop to re-order list elements.

My use case is that I have an ordered to-do list, and sometimes I want to re-order items within it.

For example, I might have something like:
```markdown
* Things to do soon
  * [ ] Fix foo
  * [ ] Do XYZ
* Things to do later
  * [ ] Make bar
    * Some additional notes about bar
* Things to do in the far future
```

I wish I could drag `Make bar` up from the "do later" list to the "do soon" list. Ideally it would also be able to move the sub-list under it along with it.

**Comment by astoilkov** [comment]
We've been considering this feature for a while now. Not only for list items but mouse-enabled workflows in the editor.

In the meantime, have you tried using Move Line Up/Down (⌥↑/⌥↓)? It moves the sub-list items as well.

**Reply by kennydo** [reply]
More mouse support in general would be great!

I've tried using Move Line Up/Down, but I haven't built into my muscle memory because 1) I'm bad with keyboard shortcuts in general and 2) I have to press it multiple times if I want to move the list item multiple lines up/down.

**Reply by astoilkov** [reply]
Yeah, I agree, the mouse interactions have their own benefits. Thanks!
